### PR TITLE
Updated permissions on shared log directories.  

### PR DESCRIPTION
**Pull request by JumaX**
… perissions changes work when multiple components are installed on a single host (Zookeeper, Broker).  Correct many yaml linting issues.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Changes:

1. Updated permissions on shared log directories for instances when multiple components are installed on the same host.  Group members will now have full access.

2. Added new molecule scenario to confirm the changes take affect: sharedhost-plaintext-rhel

3. Corrected numerous yaml linting issues.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Created a new molecule scenario: sharedhost-plaintext-rhel

**Test Configuration**:

The test does the following:

1. Adds a new Zookeeper user to the environment
2. Adds the new Zookeeper user to the logs directory
3. Restarts Zookeeper and confirms it starts correctly
4. Restarts the Broker and confirms it starts correctly 

This validates that the permissions are open enough that anyone in the confluent group can access the shared log directory.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    converge: ../../../../all.yml
+  inventory:
+    group_vars:
+      all:
+        scenario_name: sharedhost-plaintext-rhel
+verifier:
+  name: ansible
+scenario:
+  test_sequence:
+    - lint
```
Comment (domenicbove) [inline roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml:92]: what if you omit the -lint line here and then you wont need all the other changes?

File: roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  name: ansible
+  config_options:
+    defaults:
+      hash_behaviour: merge
+  playbooks:
+    converge: ../../../../all.yml
+  inventory:
+    group_vars:
+      all:
+        scenario_name: sharedhost-plaintext-rhel
```
Comment (domenicbove) [inline roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml:87]: can you also update all the log paths similar to what was done by the customer

Comment (JumaX) [reply roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml:87]: Great idea!

File: roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml (hunk 1 above)
Comment (JumaX) [reply roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml:92]: Probably not, but It's probably still good to have.

File: roles/confluent.test/molecule/sharedhost-plaintext-rhel/verify.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  tasks:
+    - import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/ksql/ksql-server.properties
+        property: security.protocol
+        expected_value: PLAINTEXT
+
+- name: Verify - control_center
```
Comment (domenicbove) [inline roles/confluent.test/molecule/sharedhost-plaintext-rhel/verify.yml:76]: Probably remove these tests, test the net new stuff

File: roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml (hunk 1 above)
Comment (domenicbove) [reply roles/confluent.test/molecule/sharedhost-plaintext-rhel/molecule.yml:92]: nvrm this is incorrect

**Comment by JumaX** [comment]
Closing for PR 406.

**Pull request closed without merging**